raiden/network/transport/matrix/client.py

```diff
@@ -325,14 +334,23 @@ def start_listener_thread(
         """
         assert not self.should_listen and self.sync_thread is None, "Already running"
         self.should_listen = True
+
         self.sync_thread = gevent.spawn(self.listen_forever, timeout_ms, exception_handler)
         self.sync_thread.name = f"GMatrixClient.listen_forever user_id:{self.user_id}"
 
+        self.message_worker = gevent.spawn(
+            self._handle_process_worker, self.response_queue, self.event_stop
+        )
```

Comment:
The relations
message_worker -> _handle_process_worker
sync_thread -> listen_forever
are not very obvious from the variable names. Can we make those more consistent?

Reply:
Do you have any ideas?

`message_handler` -> `_handle_message`
`sync_handler` -> `_loop_sync`

Reply:
I think we cannot rename `listen_forever` because it overwrites a method in the `MatrixClient`  class.

Reply:
We could do
```
def listen_forever(self):
    self._handle_sync()
```
but that does not really have a simplicity and consistency advantage.

What about:
message_worker -> _handle_message
sync_worker -> listen_forever

Reply:
That's what I did for now!